tutorials/sample_vqe_program/qiskit_runtime_vqe_program.ipynb

```diff
@@ -0,0 +1,1394 @@
+    "## Simple VQE\n",
+    "\n",
+    "VQE is an hybrid quantum-classical optimization procedure that finds the lowest eigenstate and eigenenergy of a linear system defined by a given Hamiltonian of Pauli operators.  For example, consider the following two-qubit Hamiltonian:\n",
+    "\n",
+    "\n",
+    "$$\n",
+    "H = A X_{1}\\otimes X_{0} + A Y_{1}\\otimes Y_{0} + A Z_{1}\\otimes Z_{0},\n",
+    "$$\n",
+    "\n",
+    "where $A$ is numerical coefficient and the subscripts label the qubits on which the operators act.  The zero index being farthest right is the ordering used in Qiskit.  The Pauli operators tell us which measurement basis to to use when measuring each of the qubits.\n",
+    "\n",
+    "We want to find the ground state (lowest energy state) of this Hamiltonian, and the associated eigenvector.  To do this we must start at a given initial state and iteratively vary the parameters that define this state using a classical optimizer such that the computed energies of subsequent steps are nominally lower than those previously.  The parameterized state of the system is defined by an ansatz quantum circuit that should have non-zero support in the direction of the ground state.  Because in general we do not know the solution, the choice of ansatz circuit can be highly problem specific with a form dictated by additional information. For further information about variational algorithmms, we point the reader to [Nature Reviews Physics volume 3, 625 (2021)](https://doi.org/10.1038/s42254-021-00348-9).\n",
+    "\n",
+    "\n",
+    "Thus we need at least the following inputs to create our VQE quantum program:\n",
+    "\n",
+    "1. A representation of the Hamiltonian that specifies the problem.\n",
+    "\n",
+    "\n",
+    "2. A choice of parameterized ansatz circuit, and the ability to pass configuration options, if any.\n",
+    "\n",
+    "\n",
+    "However, the following are also beneficial inputs that users might want to have:\n",
+    "\n",
+    "3. Add the ability to pass an initial state.\n",
+    "\n",
+    "\n",
+    "4. Vary the number of shots that are taken.\n",
+    "\n",
+    "\n",
+    "5. Ability to select which classical optimizer is used, and set configuraton values, if any. \n",
+    "\n",
+    "\n",
+    "6. Ability to turn on and off measurement mitigation.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "absolute-reservoir",
+   "metadata": {},
+   "source": [
+    "## Specifying the form of the input values\n",
+    "\n",
+    "All inputs to Runtime programs must be serializable objects.  That is to say that whatever you pass into a Runtime program must be able to be converted to JSON format.  Thus it is beneficial to keep inputs limited to basic data types and structures unless you have experience with custom object serialization, or they are common Qiskit types such as QuantumCircuit etc.  Fortunately, the VQE program described above can be made out of simple Python components.\n",
```

Comment:
Do you want to mention `RuntimeEncoder`? It's the default used by runtime and supports a number of types, such as complex numbers, numpy arrays, `QuantumCircuit`, and operators that are normally not JSON serializable.

Reply:
I can do that